src/ert/ensemble_evaluator/_ensemble.py

```diff
@@ -209,6 +198,7 @@ async def evaluate(
         the final result of executing all its jobs through a scheduler and driver.
         """
         self._config = config
+        outbound_event_queue = scheduler_queue or asyncio.Queue()
```

Comment:
We should refactor client, maybe not in this PR. At least to create the issue for it...